tests/integration/negative_integration_test.sh.in

```diff
@@ -24,9 +24,29 @@ fi
 # diff outputed files with reference
 cd @CMAKE_CURRENT_BINARY_DIR@/@SIM_NAME@
 
+# We convert spikes to out.dat format
+reports=@ENABLE_SONATA_REPORTS@
+if [ "$reports" = "ON" ]
+then
+  data=$(h5dump -d /spikes/timestamps -d /spikes/node_ids -y -O spikes | tr "," "\n")
```

Comment:
`h5dump ...` command exit status is not checked. It would be nice to emit an error and prevent further processing if it fails.